src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -75,11 +82,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       return;
     }
 
+    BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService();
+    Map<String, List<BlobKey>> blobs = blobstoreService.getUploads(request);
+    List<BlobKey> blobKeys = blobs.get("image");
+
     String user = userService.getCurrentUser().getEmail();
     //Use JSoup to sanitize the user data
-
```

Comment:
Is there a reason why you're undoing this change? (Someone else has changed it from none to basic, and now you're changing it back.) If it's inadvertent, please change it back.